tests/test_audit.py

```diff
@@ -93,6 +93,8 @@ def test_audit_end_to_end(session, users, groups, http_client, base_url):  # noq
     add_member(groups[groupname], users["zay@a.co"])
     add_member(groups[groupname], users["gary@a.co"])
 
+    graph.update_from_db(session)
+
```

Comment:
Yes. Prior to this change user_permissions did not rely on the graph being up to date so the test didn't need to enforce the graph being updated in order to work properly. Now however, user_permissions does need the graph to be up to date in order to provide correct information, so we need to force it to be up to date here.